### PR TITLE
questionnaire shex bug is fixed

### PR DESCRIPTION
**Pull request by rajaram5**
Bug fix for this [issue](https://github.com/ejp-rd-vp/ERN-common-data-elements/issues/24)

### CODE REVIEW

File: shex/questionnaireShape.shex [hunk 1]
```diff
@@ -9,36 +9,29 @@ PREFIX prov: <http://www.w3.org/ns/prov#>
...
-  a [obo:NCIT_C121522]
+:questionnaireShape IRI {
+  a [obo:NCIT_C121522 ncit:C130322]
 } 
 
 
 :percentageShape IRI {
   a [obo:NCIT_C25613];
-  prov:wasGeneratedBy @:qualityQuestionnaireShape;
+  prov:wasGeneratedBy @:questionnaireShape;
```
Comment (sxzhang1201) [inline shex/questionnaireShape.shex]: It seems to allow the existence of the pair ( `:percentageShape ` + `questionaireShape` WHODAS type), which is not mentioned in the figure.

Comment (rajaram5) [reply shex/questionnaireShape.shex]: [In this figure](https://www.draw.io/#G16O6jryBPCtwgrn3mC9q5mB2hflRX2Ux3) the cloud next to the person diamond says `one of two` so I understood that only one of two questionaires allowed

Comment (sxzhang1201) [reply shex/questionnaireShape.shex]: I understand that only one of two pathways allowed. So either:
`WHODAS Questionaire+ Day Score + Percentage Score + Disability`
OR:
`Quality Questionaire + Percentage Score + Total Scale Score`
But now it allows for example:
`WHODAS Questionaire + Percentage Score + Total Scale Score)

File: shex/questionnaireShape.shex [hunk 2]
```diff
@@ -9,36 +9,32 @@ PREFIX prov: <http://www.w3.org/ns/prov#>
...
   a [obo:NCBITaxon_9606];
-  obo:RO_0000056 @:qualityQuestionnaireShape;
-  obo:RO_0000056 @:whodasQuestionnaireShape
+  obo:RO_0000056 @:questionnaireShape
 }
 
-:qualityQuestionnaireShape IRI {
-  a [obo:NCIT_C121522]
+:questionnaireShape IRI {
+  a [obo:NCIT_C121522 ncit:C130322]
```
Comment (sxzhang1201) [inline shex/questionnaireShape.shex]: I found that `obo:NCIT_C130322` (http://purl.obolibrary.org/obo/NCIT_C130322) is also resolvable.  May I know the decision of these choices? 

I have encountered such a similar dilemma.

Comment (rajaram5) [reply shex/questionnaireShape.shex]: Oh no I didn't realize that two different prefixes are used for `NCIT`. But it is excellent bug finding by you :+1: . I guess the chances of having this same bug in other `ShEx` files is very likely. Can you create issue for this bug?

**Comment by rajaram5** [comment]
fixes #24

**Pull request merged**